### ISSUE TITLE
Put enum.h inside include/better-enums subdirectory.

### ISSUE THREAD

**Issue by ecorm** [aantron/better-enums#38]
You should put `enum.h` inside an `include/betterenums` subdirectory of your Git repository. This would allow superproject build scripts to clone your Git repo as-is and add `-I superproject/thirdpartylibs/betterenums/include` compiler flags where appropriate.

To include your header, we would use
```
#include <betterenums/enum.h>
```

which would avoid collisions with other headers named `enum.h`.

**Comment by aantron** [comment]
Agreed in principle, though, since it's a one-header library, it might be better to rename the file to `better_enums.h`, or something.

**Comment by ecorm** [comment]
You should at least put it inside an `include` subdirectory, so that all the other files your repository's root directory don't pollute the `#include` path. Many large project build scripts clone third-party dependencies directly from GitHub and leave the subrepo's file/directory structure intact.

**Comment by aantron** [comment]
Fair enough.
